### DISCUSSION TITLE
Subscripton: Inactive - Invoice voided

### DISCUSSION THREAD

**Discussion by Notramus** [opengisch/QField#4307] [Q&A]
Hello QField community!

We are having trouble to activate or pay for our subscription. We have added a credit card as billing information and thought that it would automatically deducted the amount from the card. But it seems that it did not do that. 

I am able to download the Invoice but there is no way to pay for it manually (#2C2C7B97-0001 <-- Invoice ID). It is marked as void. 

How can I proceed with the payment? We used the ticket system already but cannot figure out how to pay for the used amount and how to keep using QField Cloud. It would be awesome if someone could help me out :)

**Comment by herrsei123** [comment]
same problem. i'am not able to pay. after adding credit card i get a internal server error. my account is locked so i can't read or sync my project data. would be glad to get some help :)

**Comment by KOrrMap** [comment]
I also had issues with an Inactive subscription after giving payment details. I couldn't publish projects. When I opened the invoice I saw a small hyperlink that said "Pay online". When I clicked that it allowed me to accept the charges to my credit card, my account switched from "Inactive" to "Active", and I was able to publish.

**Comment by herrsei123** [comment]
Thanks for the advice. I wrote the support @ opengis.ch and they fixed my account. Now all is fine again. sincerely
